lib/pbench/server/api/__init__.py

```diff
@@ -100,6 +102,11 @@ def register_endpoints(api, app, config):
         f"{base_uri}/datasets/publish",
         resource_class_args=(config, logger),
     )
+    api.add_resource(
+        TableOfContent,
+        f"{base_uri}/datasets/contents",
+        resource_class_args=(config, logger),
+    )
```

Comment:
You changed the URI...are you intending to rename the class, as well?  (At the very least, I think Dave and I would like to see it end in a plural, but, if you're gonna change it, we might want to make a more radical change than that....  🙂)

Reply:
Sure, what would it be?

Reply:
Looking at the other `<base>/datasets/*` endpoints, I'd say the votes are 7 to 3 that it should be `DatasetsContents`.  (And, if you rename the class, you should rename the module file as well, to `datasets_contents.py`.)